courses/free-courses-en.md

```diff
@@ -750,8 +750,9 @@
 
 ### .NET
 
-* [Learn how to program: .NET](https://www.learnhowtoprogram.com/net) - Epicodus Inc.
+* [Learn how to program: C# and .NET](https://www.learnhowtoprogram.com/c-and-net) - Epicodus Inc.
```

Comment:
Removing duplicated link as Eric tell us

```suggestion
```

Is already present in C# section

Reply:
Resolved by commit: [`b106347` (#6673)](https://github.com/EbookFoundation/free-programming-books/pull/6673/commits/b106347269925bfe72e559be653461e20b4ef1b8)